data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1616 @@
+			`	He pulls out his commlink and checks a few things. "Go to Caelian and check in with Torza. He should have finished putting the team through their paces by the time you get there. See you soon, Captain!"`
+
+
+
+mission "Remnant: Cognizance 27"
+	name "Recover equipment from Baianus."
+	description "Torza has asked you to recover equipment from Baianus by <date>."
+	landing
+	blocked "Over the commlink Torza is a flurry of motion as he issues orders in several directions. 'Right now, we need another ship that can evacuate 280 tons and 20 people. Return here when you have suitable capabilities."
+	deadline 12
+	cargo "reserved" 280
+	passengers 20
+	source "Caelian"
+	destination "Baianus"
+	to offer
+		has "Remnant: Cognizance 26: done"
+	on offer
+		conversation
+			`Your commlink springs to life as you enter final approach to Caelian, revealing Torza in a flurry of motion. He focuses momentarily on the camera. "Captain <first>, and flying a ship with a substantial lifting capacity, excellent! The Korath have discovered one of our forward operating bases, and we need help evacuating it now. It is located on Baianus in the Parca system. There is 20 people and 280 cubic meters of equipment to retrieve. The Korath have setup a blockade, so you will need something reasonably fast to get through it. You can kill them or not, as you wish, but the primary task is evacuating the outpost. Embers speed your flight, Captain."`
+				accept
+	npc
+		government "Korath"
+		personality heroic plunders harvests waiting staying
+		system "Parca"
+		fleet
+			names "korath"
+			cargo 3
+			commodities Food Metal Plastic "Heavy Metals"
+			outfitters "Korath Basics" "Korath Exiles"
+			variant
+				"Korath Dredger" 1
+				"Korath Dredger (Ground Assault)" 2
+				"Korath Raider (Ember)" 6
+				"Korath Chaser" 24
+
+
+
+mission "Remnant: Cognizance 28"
+	name "Recover equipment from Baianus."
+	description "Torza has asked you to recover equipment from Baianus by <date>."
+	landing
+	blocked `The scene is one of organized chaos. One Remnant signs "We need a ship that can carry 280 cargo and 20 passengers."`
+	deadline 12
+	cargo "equipment" 280
+	passengers 20
+	source "Baianus"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 27: done"
+	on offer
+		conversation
+			`Punching through the Korath blockade finds you racing across the aurora filled skies of Baianus. You quickly slide into the designated approach track and start to enter your IFF code as a barrage of anti-air fire erupts from the ground to lance over your shoulder. As you drop into evasive maneuvers you spot the wreckage of a chaser tumbling to the ice behind you, and a firelance beam sears the air where the <ship> had just been. Moments later you get the confirmation signal that they have accepted your ID and are tracking your position.`
```

Comment:
I would add an option for immediately destroying the korath, but that's just an idea.

Reply:
It's definitely an option I will do eventually.